spring-cloud-dataflow-dependencies/pom.xml

```diff
@@ -24,7 +24,7 @@
 			<dependency>
 				<groupId>org.mariadb.jdbc</groupId>
 				<artifactId>mariadb-java-client</artifactId>
-				<version>2.2.5</version>
+				<version>${mariadb-java-client.version}</version>
```

Comment:
We can't use `mariadb-java-client.version` here as it's not defined in parent pom.

Reply:
That sort of sucks, yet it is building fine.  While I don't think we should change the parent pom here in the 1.7.x branch, maybe we should not have `spring-cloud-dependencies-parent` as the parent for `spring-cloud-dataflow-dependencies` in the master branch for 2.0?